src/ui/design/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: toc
-title: Design
+title: Design & theming
```

Comment:
**nit:** This would be better phrased as `Design & Themes` if design was intended as a noun. I would still recommend that.
```suggestion
title: Design & themes
```

Reply:
Design is intended as a verb here. It's not about the designs of your app, but about building that.